variables.tf

```diff
@@ -2,8 +2,60 @@
 # Input Variables
 ########################################################################################################################
 
-#variable "my_variable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = "default_value"
-#}
+variable "resource_group_id" {
+  description = "The Id of an existing IBM Cloud resource group where the watsonx Assistant instance will be grouped."
+  type        = string
+}
+
+variable "region" {
+  description = "IBM Cloud region where watsonx Assistant instance will be created."
+  type        = string
+}
+
+variable "tags" {
```

Comment:
suggest to call this `resource_tags` - and then also add support to `access_tags`

Reply:
done